Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupManipulators.h

```diff
@@ -63,6 +63,9 @@ namespace EMotionFX
         //! Should reset values in the manipulator mode to sensible defaults.
         virtual void ResetValues() = 0;
 
+        //! Causes values in associated property editor to refresh.
+        virtual void InvalidateEditorValues() {};
```

Comment:
nit: remove trailing `;` (also should this definitely have a default implementation or should we make it pure virtual?)

Reply:
I don't think it's necessary to force every derived class to implement this.